windows/ReactNativeCameraCPP/ReactCameraConstants.h

```diff
@@ -0,0 +1,140 @@
+#pragma once
+
+#include <functional>
+
+namespace winrt::ReactNativeCameraCPP
+{
+    static const int CameraAspectFill = 0;
+    static const int CameraAspectFit = 1;
+    static const int CameraAspectStretch = 2;
+    static const int CameraCaptureTargetMemory = 0;
+    static const int CameraCaptureTargetDisk = 1;
+    static const int CameraCaptureTargetCameraRoll = 2;
+    static const int CameraCaptureTargetTemp = 3;
+    static const int CameraOrientationAuto = UINT_MAX;
+    static const int CameraOrientationPortrait = (int)winrt::Windows::Devices::Sensors::SimpleOrientation::NotRotated;
+    static const int CameraOrientationPortraitUpsideDown = (int)winrt::Windows::Devices::Sensors::SimpleOrientation::Rotated180DegreesCounterclockwise;
+    static const int CameraOrientationLandscapeLeft = (int)winrt::Windows::Devices::Sensors::SimpleOrientation::Rotated90DegreesCounterclockwise;
+    static const int CameraOrientationLandscapeRight = (int)winrt::Windows::Devices::Sensors::SimpleOrientation::Rotated270DegreesCounterclockwise;
+    static const int CameraTypeFront = (int)winrt::Windows::Devices::Enumeration::Panel::Front;
+    static const int CameraTypeBack = (int)winrt::Windows::Devices::Enumeration::Panel::Back;
+    static const int CameraFlashModeOff = 0;
+    static const int CameraFlashModeOn = 1;
+    static const int CameraFlashModeAuto = 2;
+    static const int CameraTorchModeOff = 0;
+    static const int CameraTorchModeOn = 1;
+    static const int CameraTorchModeAuto = 2;
+    static const int CameraAutoFocusOff = 0;
+    static const int CameraAutoFocusOn = 1;
+    static const int CameraWhiteBalanceAuto = 0;
+    static const int CameraWhiteBalanceSunny = 1;
+    static const int CameraWhiteBalanceCloudy = 2;
+    static const int CameraWhiteBalanceShadow = 3;
+    static const int CameraWhiteBalanceIncandescent = 4;
+    static const int CameraWhiteBalanceFluorescent = 5;
+    static const int CameraCaptureQualityHigh = (int)winrt::Windows::Media::MediaProperties::VideoEncodingQuality::HD1080p;
+    static const int CameraCaptureQualityLow = (int)winrt::Windows::Media::MediaProperties::VideoEncodingQuality::HD720p;
+    static const int CameraCaptureQuality1080p = (int)winrt::Windows::Media::MediaProperties::VideoEncodingQuality::HD1080p;
+    static const int CameraCaptureQuality720p = (int)winrt::Windows::Media::MediaProperties::VideoEncodingQuality::HD720p;
+    static const int MediaTypeImage = 1;
+    static const int MediaTypeVideo = 2;
+
+#pragma warning( push )
+#pragma warning( disable : 4505 )

```

Comment:
I think the reason you're having to put this here is because now that you marked these static you're getting the "global/namespace  static" semantics which are different from the "class static" semantics:

class static: you don't need to have an instance to call this method
global/namespace  static: this function won't be used outside of this file (or other files that include it)

so there is probably some cpp file that includes this header, but isn't using this function - so you get a warning saying hey did you mean to do that? because nobody in that CPP file is using this function and this function is marked as a global/namespace static function, meaning nobody else will be able to call them and the code will be optimized out because of it

I think these static methods in the namespace could just be constexpr, but if that doesn't work I think what you had before the static was ok (sorry again!!)

Reply:
Scoped constants to under a class to avoid issues with global name space